### PR TITLE
feat(misc): export model & support body on claim/unclaim

### PR DESCRIPTION
**Pull request by daddykotex**
Refs MM-8185, MM-6997

### CODE REVIEW

File: src/model/model.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+export interface Timeseries {
+  key: string;
+  displayName: string;
+  description: string;
+  type: SimpleType;
+}
+
+export interface ObjectAttribute {
+  key: string;
```
Comment (blemoine) [inline src/model/model.ts]: TypeScript is structurally typed, so here `ObjectAttribute` and `OwnerAttribute` are exactly the same.
You can keep the interface separated for documentation purpose, but know that if you try to pass an `ObjectAttribute` to something expecting an `OwnerAttribute`, there will be no compilation error.

Comment (daddykotex) [inline src/model/model.ts]: Ok it's good to know, with that said, I expect to type error in javascript /s

**Comment by jtheoof** [comment]
Nice job @daddykotex quick question, the interfaces in the `model.ts` file are only here to help understand the structure of the data returned right? I don't see them used anywhere.

**Comment by jtheoof** [comment]
Also no need for internal `Refs` in public repos, instead I would link the PR link in the JIRAs themselves.

File: src/model/model.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+const basePath = '/api/v3/model';
+
+export class Model {
+  constructor(private client: Client) {
+
+  }
+
+  @authenticate
+  export(query: object): Promise<DataModel> {
```
Comment (daddykotex) [inline src/model/model.ts]: @jtheoof I use the `DataModel` here, which in turn, use all of the other interface. But it's more useful as a description really.

Comment (jtheoof) [inline src/model/model.ts]: Thanks I did not catch that, excellent !

**Comment by jtheoof** [comment]
Don't forget to update the wiki @daddykotex

**Pull request merged**